interpreter/src/main/scala/pl/touk/nussknacker/engine/variables/MetaVariables.scala

```diff
@@ -14,7 +14,7 @@ object MetaVariables {
 
   @Hidden
   def apply(metaData: MetaData): MetaVariables =
-    MetaVariables(metaData.id, properties(metaData))
+    MetaVariables(metaData.name.value, properties(metaData))
```

Comment:
Why `MetaVariables` accepts `processName: String` instead of ProcssName?

Reply:
Because `MetaVariables` is a variable used by the user in SpEL :) We don't want to force our users to use `#meta.processName.value` in every place :)

Reply:
Ahh ;)